back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/moderator_digest.rb

```diff
@@ -77,7 +77,7 @@ def generate_commands(recipient:, time: nil)
         next unless project
 
         statistics = statistics project
-        next unless nonzero_statistics? statistics
+        next unless nonzero_statistics?(statistics) && !project.admin_publication.archived?
```

Comment:
I would have a preference to turn "unless" into "if" here, as I find the `&& !archived?` a bit hard to understand with unless.

Reply:
Yeah, actually that's a good point would be more readable the other way. Will change. Cheers

Reply:
Also changed `non_zero_statistics?` to `zero_statistics?`